src/components/App.js

```diff
@@ -50,11 +56,7 @@ export default class App extends React.Component {
     }
 
     if (authError) {
-      notification.error({
-        message: 'Authentication Error',
-        description: `${authError.get('code')}: ${authError.get('description')}`,
-        duration: 0,
-      });
+      this.props.notifyAuthenticationError();
```

Comment:
`this.props.notifyAuthenticationError(authError)` surely?

Reply:
oops yes! i've been waiting for my login to expire to verify this.

Reply:
i guess i should just fake it with redux devtools. derp!